parser_library/src/context/copy_member.h

```diff
@@ -58,7 +42,25 @@ struct copy_member
             cached_definition.emplace_back(std::move(stmt));
     }
 
-    copy_member_invocation enter() { return copy_member_invocation(name, cached_definition, definition_location); }
+    // copy_member_invocation enter() { return copy_member_invocation(name, cached_definition, definition_location); }
+};
+
+// structure represents invocation of COPY member in HLASM macro library
+struct copy_member_invocation
+{
+    const id_index name;
```

Comment:
what was the reason to make these const?

Reply:
I think the only reason was that it is a structure and it does not make sense to change these members. Getter functions would be certainly better.